src/v/pandaproxy/schema_registry/error.cc

```diff
@@ -32,6 +32,8 @@ struct error_category final : std::error_category {
             return "Subject not found";
```

Comment:
> `reply_error_code::zookeeper_error is poorly named` maybe I missed when this code went it in, but does `poorly named` refer to our naming or naming that is chosen to be compatible with schema registry protocol? hopefully there is a note about that perhaps by the enum definitions.

The enum was created for pandaproxy (REST), and is used for the value of `error_code` in the response. But the error code value for schema registry has a different reason. So the enum label is not great. There is some untangling to do here, since error_reply is used in both pandaproxy and schema registry. There is indeed a comment above, but it perhaps only has enough context for me to decipher.